gradle/libs.versions.toml

```
@@ -13,6 +13,7 @@ collektive-integration = { module = "it.unibo.collektive:alchemist-incarnation-c
 
 [bundles]
 alchemist = ["alchemist", "alchemist-euclidean-geometry", "alchemist-incarnation-protelis", "alchemist-swingui"]
+collektive = ["collektive", "collektive-integration"]
```

```suggestion
collektive = ["collektive", "alchemist-collektive-incarnation"]
```

Adjust the name also at its definition